tests/unit/errors/test_common_functionality.py

```diff
@@ -346,6 +355,27 @@ def test_authz_params_info_containing_malformed_session_required_policies():
             "session_required_identities=None",
             "session_required_single_domain=None",
             "session_required_policies=None",
+            "session_required_mfa=None",
+        ],
+        ")",
+    )
+
+
+def test_authz_params_info_containing_malformed_session_required_mfa():
```

Comment:
```suggestion
def test_authz_params_info_containing_session_required_mfa():
```

I think some bad pasta got into your copypasta.

If I'm reading the test right, it's not a malformed test.

You could add a malformed test by putting in something like `{"session_required_mfa": {"foo": "bar"}}`.

Reply:
Yeah, that guy wasn't quite al dente. Good catch.